src/HomeWork3/FirstTrainingManualToEnd/FirstTrainingManualToEnd.java

```diff
@@ -0,0 +1,355 @@
+            System.out.println("Совпадений не найдено , \nцифры не равны!");
+        } else {
+            System.out.println("Совпадения есть, но не все так и плохо");
+        }
+        System.out.println("");
+        /*
+        * Task #6
+        *
+            Задание 6
+            Создайте число. Определите, является ли число трехзначным. Определите, является
+            ли его последняя цифра семеркой. Определите, является ли число четным.
+        * */
+        System.out.println("Task #6");
+        int numberT = 234;
+        char[] charsNumberT = String.valueOf(numberT).toCharArray();
+        if (charsNumberT.length < 3) {
+            System.out.println("Целое число " + numberT + " не трехзначное");
+        } else {
+            System.out.println("Целое число " + numberT + " трехзначное");
+            if (charsNumberT[2] == 7) {
+                System.out.println("Последняя цифра равна 7.");
+            } else {
+                System.out.println("Последняя цифра не равна 7.");
+            }
+        }
+        System.out.println("");
+        /*
+        * Task #7
+        * Имеется прямоугольное отверстие размерами a и b, определить, можно ли его
+          полностью закрыть круглой картонкой радиусом r.
+        * */
+        System.out.println("Task #7");
+        int sideA = 12;
+        int sideB = 3;
+        int radius = 45;
+//        int radiusAb = (sideA * sideA + sideB * sideA) / 4; //45
+//        System.out.println(radiusAb);
+        if (radius >= (sideA * sideA + sideB * sideA) / 4) {
+            System.out.println("Закрыть можно, но осторожно");
+        } else {
+            System.out.println("К сожелению не все так просто");
+        }
+        System.out.println("");
+        /*
+        * Task #8
+        * Имеется целове число (задать с помощью Random rand = new Random(); int x =
+          rand.nextInt() ). Это число – количесво денег в рублях. Вывести это число, добавив к
+          нему слово «рублей» в правильном падеже.
+        * */
+        System.out.println("Task #8");
+        Random random = new Random();
+        int x = random.nextInt(28);
+        if (x == 1) {
+            System.out.println(x + " рубль");
+        } else if (x > 1 && x <= 4) {
+            System.out.println(x + " рубля");
+        } else if (x > 4 && x < 28) {
+            System.out.println(x + " рублей");
+        }
+        System.out.println("");
+        /*
+        * Task #9
+        *
+            Задание 9
+            Изменить пример с суммой чисел таким образом, чтобы рассчитывалась не сумма, а
+            произведение, т.е. факториал числа.
+        * */
+        System.out.println("Task #9");
+//?????????????????????????????????????????????????????????????????????????????????????????????????????????????????????
+        double sumNumbers = 1;
+        int randomNumbers = (int) (Math.random() * 20);
+        int count = 0;
+
+        while (randomNumbers != 0) {
```

Comment:
it is not factorial. we get number and need in result 1*2*...*randomNumbers. p.s. for a single value, use a single number in the name (randomNumber, not randomNumbers)